settings.py

```diff
@@ -371,6 +371,7 @@
     'custom.nic_compliance',
     'custom.hki',
     'custom.champ',
+    'custom.covid',
```

Comment:
Is this so it can find the management command?

Reply:
yes